torch/onnx/_internal/fx/export_options.py

```diff
@@ -0,0 +1,33 @@
+"""Options for FX exporter static kwargs."""
```

Comment:
```suggestion
"""Options for FX exporter."""
```
I wonder what 'static kwargs' implies/adds...